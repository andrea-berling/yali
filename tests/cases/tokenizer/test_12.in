("bar"+"baz") != "other_string"
