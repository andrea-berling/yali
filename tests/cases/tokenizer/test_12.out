LEFT_PAREN ( null
STRING "bar" bar
PLUS + null
STRING "baz" baz
RIGHT_PAREN ) null
BANG_EQUAL != null
STRING "other_string" other_string
EOF  null
